src/include/cart/api.h

```diff
@@ -716,7 +715,7 @@ crt_ep_abort(crt_endpoint_t *ep);
 	BOOST_PP_IF(BOOST_PP_SEQ_SIZE(fields_in),			\
 		CRT_GEN_PROC_FUNC(rpc_name##_in, fields_in), )		\
 	BOOST_PP_IF(BOOST_PP_SEQ_SIZE(fields_out),			\
-		CRT_GEN_PROC_FUNC(rpc_name##_out, fields_out), )	\
+		CRT_GEN_PROC_FUNC(rpc_name##_out, fields_out), )		\
```

Comment:
(style)  line over 80 characters
(style)  space prohibited before that close parenthesis ')'